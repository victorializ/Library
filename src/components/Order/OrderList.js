import React  from 'react';

import { Order } from './Order';
import { ItemsList } from '../ItemsList';

const OrdersList = ({data}) => <ItemsList data={data} Component={Order} />

export { OrdersList };