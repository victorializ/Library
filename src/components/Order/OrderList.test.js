import React from 'react';
import { shallow } from 'enzyme';

import { OrdersList } from './OrderList';

describe('OrdersList component', () => {
    it('renders correctly', () => {
        expect(shallow(<OrdersList />)).toMatchSnapshot();
    }); 
});