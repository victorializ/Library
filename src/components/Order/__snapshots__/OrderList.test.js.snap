// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersList component renders correctly 1`] = `
<ItemsList
  Component={[Function]}
/>
`;
